peg/Parser.hx

```diff
@@ -207,17 +211,30 @@ class Parser {
 		return name;
 	}
 
+	function mapParsedType(type:String):PType {
+		return switch(type) {
+			case 'int' | 'Int' | 'integer' | 'Integer': TInt;
+			case 'float' | 'Float': TFloat;
+			case 'string' | 'String': TString;
+			case 'string[]' | 'String[]': TArrayOfString;
```

Comment:
These and similar (e.g. `int[]`) should be parsed as `TArray(TString)`

Reply:
done in 09f1f4c - I've implemented a recursion to handle this.